presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -511,6 +511,25 @@ dropping the table from the metadata catalog using ``TRUNCATE TABLE``.
     -----------+------+-----------+---------
     (0 rows)
 
+DELETE
+^^^^^^^^
+
+The iceberg connector can delete data in one or more entire partitions from tables by using ``DELETE FROM TABLE``::
```

Comment:
```suggestion
The iceberg connector can delete data in one or more entire partitions from tables by using ``DELETE FROM``::
```

``DELETE FROM TABLE`` implies that those three words are required. Also consider adding a following sentence introducing the example and the example table name lineitem, something like 

"For example, to delete from the table ``lineitem``:  "